src/App.vue

```diff
@@ -24,11 +24,11 @@
   const videoSet = waffleCone(ContentSource.Videos, 2)
   const articleSetForSingleSource = waffleCone(ContentSource.Articles, 6)
   
-  var row1: number = articleSetForSingleSource.slice(0, 1)
-  var row2: number = articleSetForSingleSource.slice(2, 3)
-  var row3: number = articleSetForSingleSource.slice(3, 4)
-  var row4: number = articleSetForSingleSource.slice(4, 5)
-  var row5: number = articleSetForSingleSource.slice(5, 6)
+  var row1: number = articleSetForSingleSource.slice(0, 1)[0]
+  var row2: number = articleSetForSingleSource.slice(2, 3)[0]
+  var row3: number = articleSetForSingleSource.slice(3, 4)[0]
+  var row4: number = articleSetForSingleSource.slice(4, 5)[0]
+  var row5: number = articleSetForSingleSource.slice(5, 6)[0]
```

Comment:
Just setting an index of 0 for now to make it not an array?

Reply:
It’s more that waffle works with both multi and single cards. So the single cards expect a single number. Eventually I’ll probably just change the single cards to expect an array of one